src/engine/cuecontrol.cpp

```diff
@@ -854,15 +832,15 @@ void CueControl::cueDefault(double v) {
 void CueControl::pause(double v) {
     QMutexLocker lock(&m_mutex);
     //qDebug() << "CueControl::pause()" << v;
-    if (v != 0.0) {
+    if (v != 0) {
```

Comment:
I don't think I understand, do you mean you were hoping this would do an integer comparison?

BTW I like this website for looking at (and being able to share) the results of compiling with clang/gcc: https://godbolt.org/g/gR11iQ

Anyway, no big deal -- for readability I like the types of the literals to match so you have a hint for what you're dealing with.